packages/database/src/migrations/20210303025836-AddFETPGradAreaOfExpertiseOverlays-modifies-data.js

```diff
@@ -24,14 +24,12 @@ const BASE_OVERLAY = {
   userGroup: 'FETP Graduates',
   isDataRegional: true,
   measureBuilderConfig: {
-    aggregationType: 'SUM',
     entityAggregation: {
-      aggregationType: 'REPLACE_ORG_UNIT_WITH_ORG_GROUP',
       dataSourceEntityType: 'individual',
       aggregationEntityType: 'district',
```

Comment:
Very minor but I think if we're removing `aggregationType` then we no longer need `aggregationEntityType` as well

Reply:
Thanks @chris-bes! Oddly it seems to be required even with the `aggregationType` removed - the overlays don't work properly without it (or at least they don't work properly without it the way I've currently set them up)

Reply:
Ahh sorry, you're right. I didn't realize that this overlay is shaded polygons by districts. All good!